src/Api/Startup.cs

```diff
@@ -133,6 +134,7 @@ public void ConfigureServices(IServiceCollection services)
 
         // Services
         services.AddBaseServices(globalSettings);
+        services.AddToolsServices();
```

Comment:
💭 Just happened upon this PR -- can this move lower down, before some of the more core / critical DI?